src/bosh-director/lib/cloud/dummy.rb

```diff
@@ -363,6 +368,12 @@ def spawn_agent_process(agent_id, legacy_agent_path = nil)
 
       private
 
+      def method_missing(method, *args, &block)
+        @outer.send(method, *args, &block)
```

Comment:
`@outer` is not defined anywhere, nor is it assigned from anywhere that I can find in this PR. Is there something I don't understand? 
Why is method missing overridden?